src/Minefield/Game/Coordinate.cpp

```diff
@@ -1,25 +1,26 @@
 #include <Minefield/Game/Coordinate.h>
+
 #include <regex>
 #include <sstream>
 
 namespace game
 {
-    std::vector<Coordinate> parseCoordinates(std::string const &input)
-    {
-        std::vector<Coordinate> coordinates;
-        std::istringstream iss(input);
-        std::string pair;
+std::vector<Coordinate> parseCoordinates(std::string const& input)
+{
+    std::vector<Coordinate> coordinates;
+    std::istringstream iss(input);
+    std::string pair;
 
-        while (iss >> pair)
+    while (iss >> pair)
+    {
+        size_t comma_pos = pair.find(',');
```

Comment:
#should be consistent with your coding convention; if following what you've been doing so far, this should read `commaPos`